lighthouse-core/report/v2/report-styles.css

```diff
@@ -36,19 +36,19 @@
   --lh-category-score-width: 50px;
 }
 
-* {
+.lh-root * {
   box-sizing: border-box;
 }
 
-body {
+body.lh-root {
```

Comment:
in devtools case this won't be a body tag, so i'd favor doing just tagless `.lh-root`